src/test/java/com/surveyMonkey/ViewTest.java

```diff
@@ -106,4 +107,22 @@ public void testCorrectPassword() throws IOException {
         surveyRepository.delete(fts.getTestSurvey());
     }
 
+//    @Test
```

Comment:
Please delete the code, don't leave it in as comments

Reply:
Alright I will do that now, thanks for pointing this out